Rakefile

```diff
@@ -22,6 +22,17 @@ task :test do
   Rake::Task['test:core'].invoke
 end
 
+begin
+  require 'ci/reporter/rake/test_unit'
```

Comment:
This gem uses minitest, not test_unit. That's probably why it doesn't report anything in Jenkins.

Reply:
Updated to `minitest-reporters` and Jekins passing all the check here https://github.com/theforeman/smart_proxy_dynflow/pull/119

Reply:
@ekohl i've used [minitest-junit](https://github.com/aespinosa/minitest-junit) there to create xml report.

Reply:
The reason I suggested `minitest-reporters` is that we already use it in other gems and have integrated it in the pipeline: https://github.com/theforeman/jenkins-jobs/blob/2cb54780103b7e194ed88ce11221941a728c65dd/theforeman.org/pipelines/test/smart-proxy.groovy#L98-L100

With minitest-junit (which I didn't know about) I don't see a way to change behavior via environment variables so that would require additional work somehow.

My preference would be to stick to `minitest-reporters` unless there's a good reason to switch. If we do switch, we should switch all Smart Proxy plugins.